jobsdb/jobsdb_migration_checkpoints.go

```diff
@@ -100,12 +100,15 @@ func (jd *HandleT) CheckpointInTxn(txHandler transactionHandler, migrationCheckp
 	var mcID int64
 
 	stmt, err = txHandler.Prepare(sqlStatement)
-	// skipcq: SCC-SA5001
-	defer stmt.Close()
 	if err != nil {
 		return mcID, err
 	}
 
+	defer func() {
+		err := stmt.Close()
+		jd.logger.Warnf("Error closing statement: %v", err)
```

Comment:
we appear to be overly confident here that the statement will never manage to close successfully...!

Reply:
Good catch Aris

Reply:
Nice catch :joy: